scRNA-seq-advanced/setup/SCPCP000005-samples/Snakefile

```diff
@@ -0,0 +1,30 @@
+# Workflow for preprocessing SCPCP000005 data used for integration instruction
+# Raw files are expected to be present in the `config[raw_dir]` directory
+
+configfile: "config.yaml"
+outdir = "processed"
+
+rule target:
+  input:
+    expand(os.path.join(outdir, "{library}.rds"),
```

Comment:
With outdir now not including the `processed` directory we need to add that.
(I'm just putting it in the quoted string because snakemake only works on posix systems, so `os.path.join` isn't as necessary to use religiously)
```suggestion
    expand(os.path.join(outdir, "processed/{library}.rds"),
```